<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewAccountClass</class>
 <widget class="QDialog" name="DialogNewAccountClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>407</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DialogNewAccount</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label_AccountPlatform">
        <property name="text">
         <string>账户所属软件或平台名：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ledit_accountPlatform"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAccount">
        <property name="text">
         <string>账号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ledit_account"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gBox_extraInformation">
     <property name="title">
      <string>包含的附加信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cbox_phoneNumber">
        <property name="text">
         <string>手机号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="ledit_phoneNumber"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbox_user">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="ledit_user"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gBox_thePasswordProperty">
     <property name="title">
      <string>选择生成密码的格式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_passwordLength">
        <property name="text">
         <string>密码位数：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_passwordLengthMaximum">
          <property name="text">
           <string>最大位数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_passwordLengthMaximum"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_passwordLengthMinimum">
          <property name="text">
           <string>最小位数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_passwordLengthMinimum"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_haveSpecialSymbols">
          <property name="text">
           <string>是否包含特殊符号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_haveSpecalSymbols">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_haveUpperAndLowerCase">
          <property name="text">
           <string>是否含有大小写：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_haveUpperAndLowerCase">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_isIrreversible">
          <property name="text">
           <string>是否进行不可逆加密：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_isIrreversible">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pbtn_ok">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbtn_cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DialogNewAccount.qrc"/>
 </resources>
 <connections/>
</ui>
